src/components/AssetSelection/AssetSelection.tsx

```diff
@@ -18,6 +18,7 @@ type TradeAssetSelectBaseProps = {
   assetIds?: AssetId[]
   isLoading?: boolean
   buttonProps?: ButtonProps
+  onlyConnectedChains: boolean
```

Comment:
Do we want to make this optional? Seems like regardless of the intent, that would make more sense.
i.e:
- If the intent is always to have this set to true (as it seems to be the case here) except for the odd cases when we don't, then we could `= true` it when destructuring it
- Or the other way around, if the intent is to assume a false default to ensure we don't change behavior in places where we shouldn't, then ommitting it would evaluate to falsy 

Assuming the former because of the positive vs. negative semantics here, as well as consumptions

Reply:
The intent is to be explicit, as the UI behaviour isnt really obvious here. The optional flag in the selector is an artefact of the design and is a cause of the `assetId ?? ''` horror we see everywhere